GuildedRoseLLC/Items/ItemsViewController.swift

```diff
@@ -44,6 +43,19 @@ public class ItemsViewController: UIViewController, UICollectionViewDelegate {
         
         itemCollectionView.dataSource = dataSource
         itemCollectionView.delegate = self
-        
+    }
+    
+    public override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "DetailsPageSegue" {
+            let destination = segue.destination as! DetailViewController
+            let button = sender as! UIButton
+            let item: Item = (self.dataSource?.items[button.tag])!
+            destination.setItem = {
+                () in
+                print(item)
```

Comment:
Request for Change: Can you get rid of this print statement?

Reply:
removed